spdm_dump/spdm/spdm_dump_spdm.c

```diff
@@ -2458,6 +2466,9 @@ void dump_spdm_psk_finish_rsp(const void *buffer, size_t buffer_size)
     libspdm_set_data(m_spdm_context, LIBSPDM_DATA_SESSION_USE_PSK, &parameter,
                      &use_psk, sizeof(use_psk));
 
+    /*use next key when next seesion start*/
+    m_current_psk_key_index++;
```

Comment:
What if it > LIBSPDM_MAX_SESSION_NUM ?

Reply:
Thanks. I have added the check in use key caller function `spdm_dump_session_data_provision`: 
```
    if ((m_current_dhe_key_index >= LIBSPDM_MAX_SESSION_NUM) ||
        (m_current_psk_key_index >= LIBSPDM_MAX_SESSION_NUM)) {
        LIBSPDM_ASSERT(false);
        return LIBSPDM_STATUS_INVALID_STATE_LOCAL;
    }
```

Reply:
I dont think ++ should be separated from MAX value check.

They should be in same function.